home-assistant-addon-dev/CHANGELOG.md

```
@@ -11,10 +11,15 @@
 
 # What's Changed
 
+> [!WARNING]
+> **`HmIP-RFUSB-TK` issues**
+>
+> If you are using a `HmIP-RFUSB-TK` (with `-TK` in the product name) to communicate with your HomematicIP devices, please make sure to update to Home Assistant OS [17.1rc1](https://github.com/home-assistant/operating-system/releases/tag/17.1.rc1) or newer first, because only with 17.1 or newer full compatibility with `HmIP-RFUSB-TK` is restored!
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)